Fix footer not sticky on the bottom

lindotex opened the pull request:
`#` ✨ Codu Pull Request 💻

Fixes #1171 
<!-- for example `Fixes #1012` -->

## Pull Request details
- Added an min height to the post component. (tiny small change), because I didn't want to make any huge alterations on the code.

## Any Breaking changes

- None

## Associated Screenshots

Desktop:
<img width="1437" alt="desktop view of the page" src="https://github.com/user-attachments/assets/e28397ca-348b-495d-adb7-ea123f73bf94">

Mobile:
<img width="375" alt="mobile view of the page" src="https://github.com/user-attachments/assets/98d8fcc3-dfcb-4ab1-92f7-a08c6705131c">

lindotex committed:
fix: footer
added an new className "min-h-96" to the div where the post tabs are located.

lindotex committed:
fix: Footer didn't stick to the bottom.

I added an extra className "min-h-96" on the div
containing the post tabs.That made the footer goes
to the bottom of the page. I tried to make the
Footer get Sticked to the bottom, using an
"sticky bottom-0" on the Footer, but it didn't
work. The component must be flex.

NiallJoeMaher reviewed (changes requested):
The env should not be deleted and the min-h-96 might be too high or low depending on the screen size.

lindotex committed:
fix: Footer sticking at the bottom of the page

added an grid behavior on the layout, specifying that the children component must fill the available space. That way, the footer will be always placed at the bottom.

lindotex committed:
Merge branch 'develop' into fix-footer-not-sticky-on-the-bottom

NiallJoeMaher committed:
Merge branch 'develop' into fix-footer-not-sticky-on-the-bottom

NiallJoeMaher commented:
Just need `npm run prettier:fix`

NiallJoeMaher commented on app/(app)/layout.tsx:
```diff
@@ -79,13 +79,15 @@ export default async function RootLayout({
 
   return (
     <>
```
No longer need the fragments. Let me know if you have time to update this.

NiallJoeMaher commented:
You can reopen if you decide to tackle again.

lindotex commented:
Sorry by my late answer.
I was Traveling 😅...
I have the code ready.

Zaid-B24 commented:
I can work on this issue, if no one has any problem with it.
